.eslintrc.js

```diff
@@ -1,13 +1,6 @@
 module.exports = {
```

Comment:
We should add these bulk fix commits to [`.git-blame-ignore-revs`](https://github.com/matrix-org/matrix-js-sdk/blob/193c38523c6304c86e56d6bdc5089ece26ad34da/.git-blame-ignore-revs) so we can easily skip over them in the git blame interface

(same with other bulk Typescript fixes that have happened recently)

Reply:
Yes, definitively. I will open PRs later :+1: